### PR TITLE
Milo - Consolidate docs to single document

### LINKED ISSUE #22: Change documentation order to assemble electronics and cross brace into table before mounting rest of the mill
**Describe the bug**
There is an issue with the order of assembly. Electronics table should probably come before the mounting table to rest of assembly. Or a least the cross brace be added to the table assembly section.

**Document Name / Location**
table assembly

**Required Resolution**
Electronics table should probably come before the mounting table to rest of assembly. Or a least the cross brace be added to the table assembly section.

**Screenshots**

**Additional context**

### LINKED ISSUE #24: FMJ Docs call for M5x10 but the text asks for M5x12
**Describe the bug**
In the FMJ Docs the 'Components Required' box calls for M5x10 but the text in the instruction says to use M5x12.

**Document Name / Location**
fmj assembly page.

**Required Resolution**
Consistent callout for bolt - suggest M5x10 to match LDO kit.

### PR DESCRIPTION
**Pull request by EdwardChamberlain**
Major restructure of manual.
 - Combine electronics and assembly manual 
 - Re-stage assembly box build to assemble electronics before fixing to Y axis

Minor QOL updates
- Add rail lengths
- SPAG
- Added additional imagery
- Add GitHub Links
- Numbered files for structure
- Restructured images

### CODE REVIEW

File: docs/milo/manual/chapters/40_y_axis_assembly.md
Comment (dmason1992) [inline docs/milo/manual/chapters/40_y_axis_assembly.md:1]: I'd like to see the Rei method documented somewhere other than a random discord thread. Not going to hold up this PR for it but worth noting for the future.

File: docs/milo/manual/chapters/60_z_axis_assembly.md
Comment (dmason1992) [inline docs/milo/manual/chapters/60_z_axis_assembly.md:1]: Add note that the lengths shown are for a standard 60mm clearance build. A 120mm clearance build will have A+B be +60mm in length

File: docs/milo/manual/chapters/70_fmj_assembly.md
Comment (dmason1992) [inline docs/milo/manual/chapters/70_fmj_assembly.md:1]: Add same note to component required box about extrusions A and B as per text above

File: docs/milo/manual/chapters/60_z_axis_assembly.md
Comment (EdwardChamberlain) [reply docs/milo/manual/chapters/60_z_axis_assembly.md:1]: Added note

File: docs/milo/manual/chapters/80_table_assembly.md
Comment (dmason1992) [inline docs/milo/manual/chapters/80_table_assembly.md:1]: Add section for the small cross brace to be added at this stage just to round off the mechanical build rather than sneaking it in to the electronics. People won't be able to resist popping the mill on the top of the table at this point.

File: docs/milo/manual/chapters/70_fmj_assembly.md
Comment (EdwardChamberlain) [reply docs/milo/manual/chapters/70_fmj_assembly.md:1]: Added

File: docs/milo/manual/img/assemble_electronics/assemble_electronics_step_0.png
Comment (dmason1992) [inline docs/milo/manual/img/assemble_electronics/assemble_electronics_step_0.png:1]: Move images for installing cross brace from here to table build

File: docs/milo/manual/img/assemble_electronics/assemble_electronics_step_1.png
Comment (dmason1992) [inline docs/milo/manual/img/assemble_electronics/assemble_electronics_step_1.png:1]: Add tip to ensure SD card is installed before attachign board to table as it's not easy to install after.

**Review by dmason1992 — CHANGES_REQUESTED** [review]
Please address changes requested in comments before merging.

Otherwise fantastic work @EdwardChamberlain !

File: docs/milo/manual/chapters/80_table_assembly.md
Comment (EdwardChamberlain) [reply docs/milo/manual/chapters/80_table_assembly.md:1]: Moved

Comment (EdwardChamberlain) [reply docs/milo/manual/chapters/80_table_assembly.md:1]: Note added below this image

**Review by dmason1992 — APPROVED** [review]

**Pull request merged**
